bin/bst-init.ts

```diff
@@ -70,7 +71,7 @@ const questions = [
 ];
 
 program
-    .description("setup example project and configuration")
+    .description("Setup example project and configuration")
     .action(() => {
         console.log(chalk.yellow("Welcome to the bespoken tools CLI."));
```

Comment:
Welcome to the Bespoken CLI

Reply:
https://github.com/bespoken/bst/pull/619